BasicSample/Assets/Editor/MixedRealitySampleQuickSetupWindow.cs

```diff
@@ -104,17 +104,23 @@ private void OnGUI()
             m_selectedMRConfiguration = MixedRealityProjectConfiguration.None;
             EditorGUIUtility.labelWidth = Default_Label_Width;
             GUIStyle titleStyle = new GUIStyle(EditorStyles.largeLabel) { fontStyle = FontStyle.Bold };
-            GUIStyle contentStyle = new GUIStyle(EditorStyles.label) { fontStyle = FontStyle.Bold };
+            GUIStyle headingStyle = new GUIStyle(EditorStyles.label) { fontStyle = FontStyle.Bold };
+            GUIStyle contentStyle = new GUIStyle(GUI.skin.box);
+            if(EditorGUIUtility.isProSkin)
+            {
+                contentStyle.normal.textColor = Color.white;
```

Comment:
nit: I think Unity uses a light grey instead of pure white in that theme, but more importantly I think there's a way automatically re-use Unity's built in styles

is it possible to get the color here out of EditorStyles.label.onNormal.textColor?

Reply:
let me try if that works. I have tried several ways to make this work, but this seemed the clean and easy path.